components/Actuators/HighLevel/hopperMotor.py

```diff
@@ -0,0 +1,137 @@
+import logging as log
+
+from magicbot import feedback, tunable
+from utils.DirectionEnums import Direction
+from components.Input.breakSensors import Sensors, State
+
+class HopperMotor:
+    """
+    Allows you to run motors in the hopper
+    """
+    compatString = ["teapot"]
+
+    motors_hopper: dict
+    sensors: Sensors
+    intakeSpeed = tunable(.1)
+    movingSpeed = tunable(.08)
+
+    def setup(self):
+        """
+        Sets up hopper motors
+        """
+        self.hopperSpeedFore = 0

```

Comment:
Future change and use just hopperSpeedFore == 0.0 and remove the bool, same for Back.

Reply:
and places checking for it running can call get the getter that knows 0 means not running